### PR TITLE
Change organisation ref abstract

### PR DESCRIPTION
**Pull request by skinkie**
This does the trick in master **with** backwards compatibility and an example to validate for the backwards compatibility. So master it is.

### CODE REVIEW

**Comment by ue71603** [comment]
@duexw  What do you think?

**Review by trurlurl — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

File: xsd/NeTEx_publication_timetable.xsd [hunk 1]
```diff
@@ -1285,17 +1285,6 @@ Provides a general purose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
-		<!-- =====Organisation============================== -->
```
Comment (skinkie) [inline xsd/NeTEx_publication_timetable.xsd:1288]: Investigate if this deletion actually is correct. I remember undoing it.

Comment (ue71603) [reply xsd/NeTEx_publication_timetable.xsd:1288]: it seems to me that it is is still in the file on line 1376ff

**Comment by ue71603** [comment]
@Aurige  pls review

**Comment by Aurige** [comment]
looks ok :-)

**Review by Aurige — APPROVED** [review]

**Pull request merged**